tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -77,3 +83,37 @@ def test_delete_ticket_holder_with_valid_order_id(self):
             delete_ticket_holders_no_order_id()
             ticket_holder = TicketHolder.query.get(attendee_id)
             self.assertIsNot(ticket_holder, None)
+
+    def test_send_monthly_invoice(self):
+        """Method to test marking of event invoices as due"""
+
+        with self.app.test_request_context():
+            # ticket fee factory
+            ticket_fee_test = TicketFeesFactory()
+            # event factory 
+            test_event = EventFactoryBasic(
+                state='published'
+            )
+            test_event.tickets_sold = 1
+            # user factory
+            test_user = UserFactory()
+            # order factory  
+            test_order = OrderFactory(
+                status = 'completed'
+            )
+            test_order.completed_at = datetime.datetime.now() - datetime.timedelta(days=30)
+            test_order.amount = 100
+            test_order.event = test_event
+            # ticket holder factory
+            test_ticket_holder = TicketHolderFactory()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'test_ticket_holder' is assigned to but never used](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4928052)